site/content/en/docs/enterprise/subscription-managment.md

```diff
@@ -64,37 +75,136 @@ All limits of the **Free** plan will be removed.
+
+Here’s how you can do it:
+
+1. Sign up for a CVAT.ai account and log in.
+2. (Optional) If you add the VAT/tax number to the organization,
+   first create an [organization](/docs/manual/advanced/organization/) and
+   switch to an **Organization** account.
+3. Navigate to the top right corner, next to the nickname,
+   click on the arrow > uprate to the plan.
+4. Switch on the **I would like the invoice to include additional
+   data (address, phone number, VAT information)** toggle and click **Get Started**.
+
+   ![Stripe Link](/images/add-invoice.jpg)
+
+5. You will see the billing page:
+
+   ![Stripe Link](/images/add-vat.jpg)
+
+   - **Billing Address**: Enter the billing address you want to appear on the invoice
+     in the address field **(1)**.
+   - **VAT Information** and **Business Name**: Select the checkbox **I am purchasing as a business** and enter
+     your VAT and business name information **(2)**.
+
+6. Select checkbox **I agree to refund policy** **(3)**.
+
+7. Click **Pay & Subscribe**.
+
+All information you've added will appear on the billing page and in the invoice.
+
+![Stripe Payment Info](/images/sm-profile-upd-inf.png)
+
+By following these steps, you can seamlessly add VAT and other crucial information to your
+invoices, making your financial transactions with CVAT.ai transparent and compliant.
+
+### How to add/ change VAT/tax and other information to the CVAT.ai invoice after the first payment?
+
+In the top right corner, near the nickname, click on the arrow > manage plan.
+
+You will see the Stripe page. Go to the **Billing Information** > **Update Information**.
+
+![Stripe Payment Info](/images/sm-profile-upd-inf-2.png)
+
+### Can a paid invoice be modified?
+
+Once an invoice has been paid, it is not possible to modify it. This restriction is due to the
+limitations of the payment processing platform used, which in the case of CVAT.ai, is [Stripe](https://stripe.com/).
+
+Stripe's policy dictates that revisions to an invoice can only be made before payment.
+For more comprehensive information on this policy, please refer to Stripe's official documentation
+on [invoice edits](https://docs.stripe.com/invoicing/invoice-edits#:~:text=Stripe%20lets%20you%20revise%20a,Edit%20the%20invoice%20description.)
+at their website.
+
+### How can I get a quote before I subscribe? How to add a PO number to my invoices?
+
+If you require a quote from CVAT for payment via bank transfer, certain criteria must be met:
+
+- The **total subscription cost must be $396 and up**.
+- Quotes are available exclusively **for annual subscriptions**.
+
+Should you meet these requirements, please write to [support@cvat.ai](mailto:support@cvat.ai)
+
+### Can you sign an agreement before I subscribe?
+
+Sign of specific agreements and approvals are available if you meet specific criteria
+(the **total subscription cost must be $10,000 and up**), for more details contact [support@cvat.ai](mailto:support@cvat.ai)
```

Comment:
```suggestion
(the **total subscription cost must be $10,000 and up per year**), for more details contact [support@cvat.ai](mailto:support@cvat.ai)
```

Reply:
Also, only available for annual subscription? 
Otherwise we can't guarantee that users will not unsubscribe after couple of months with ~850$ per month subscription. 

@nmanovic